templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/modeling_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -21,6 +21,7 @@
 import os
 
 import torch
+import torch.nn.functional as F
```

Comment:
So this one is no longer needed, is it?

Reply:
No, it shouldn't be needed anymore, I was wondering if we should keep it or not.

Reply:
probably remove it then if tests pass.

We are going to get rid of `F` after we merge this - will use `nn.functional` everywhere consistently.